db/migrate/20150223184008_add_defaults_to_user.rb

```diff
@@ -0,0 +1,6 @@
+class AddDefaultsToUser < ActiveRecord::Migration
+  def change
+    change_column_default(:users, :status, 'pending')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.